### ISSUE TITLE
Feat: `ruff --list-pylint-coverage` to get already implemented Pylint rules

### ISSUE THREAD

**Issue by tuttle** [astral-sh/ruff#22942]
I understand that Ruff aims to eventually cover reasonable set of Pylint rules. However, Pylint is still pretty sharp and provides valuable insights. I would like to use Pylint in my project **only for the rules that Ruff doesn't support yet**.

I intend to switch off all the rules supported by Ruff in `pylintrc` (the list appears to be too long to maintain it in `pyproject.toml`).

Using the description from issue #970 and the **Edits** pulldown, I did this:
- extracted a list,
- applied the last diffs to the text,
- filtered out items that were checked,
- filtered out items that are struck through (here I'm not certain),

and transformed them using this command:

```bash
grep -Ev '\[ \]|~|^#' pylint | tr -d '`' | awk '{ 
    gsub(/[()]/, "", $6); 
    printf "    \"%s\":  # %s%-*s- covered by ruff in %s\n", $5, $3, (50 - length($3) - 10), " ", ($6 != "" ? $6 : "?") 
}' | sort
```

Getting this format for the `disable` list od 228 lines:

```
    "C0103":  # invalid-name                            - covered by ruff in N815
    "C0105":  # typevar-name-incorrect-variance         - covered by ruff in PLC0105
    "C0112":  # empty-docstring                         - covered by ruff in D419
    "C0113":  # unneeded-not                            - covered by ruff in SIM208
    ....
```

It occurred to me that **Ruff could potentially generate a similar list directly**. It would be fantastic if updating the `disable` list in a `pylintrc` (or `pyproject.toml`) could be made this convenient via a built-in Ruff command.

**Comment by ntBre** [comment]
Thanks for the suggestion and for the neat script gathering this data! This seems a bit niche to me to warrant a CLI flag, but perhaps we could document the rule correspondence somewhere more accessible, or at least update the description in #970 to help.

**Comment by vintitres** [comment]
> Getting this format for the disable list (228 lines)


Note that some of the ruff rules listed in #970 are in preview mode, so unless you run ruff in preview mode the `disable` list will disable some rules that `ruff check` doesn't catch.

I used `ruff check --show-settings | sed -n '/linter.rules.enabled = \[/,/\]/p' | awk '{print $NF}' | sed 's/[(),]//g' | sort -u` to list the ones that my `ruff` setup and version actually checks, and ran that against the list of pylint ones.

**Comment by tuttle** [comment]
@ntBre @vintitres Thank you both.

Good point regarding the Preview mode. However, since we haven't had any issues with Ruff's preview rules so far, we’ve kept the mode enabled in our project, hence there is no conflict here, rather by accident. :-)

**Comment by tuttle** [comment]
Sorry for being off topic.

@vintitres It's a very neat `sed` trick you used to rip out the `linter.rules.enabled` section! Thanks for sharing. :-)

Additionally, I found out `awk '{print $NF}' | sed 's/[(),]//g'` could in this case be replaced by `grep -oP '\(\K[^)]+'`:

- `-o:` Tells grep to only output the matching part of the line.
- `-P`: Enables Perl-compatible regular expressions.
- `\(`: Matches the opening parenthesis.
- `\K`: A "keep out" marker—it tells grep to ignore everything it matched up to this point in the final output.
- `[^)]+`: Matches one or more characters that are not a closing parenthesis.

Adding both to my toolbox.
